src/tests/containers/ProjectsList.test.js

```diff
@@ -26,4 +26,12 @@ describe('ProjectsList', () => {
   it('should have a header Volunteers Directory', () => {
     expect(wrapper.find('Header').text()).toBe('List of Projects')
   })
+
+  it('should have a Paginate component', () => {
```

Comment:
These 2 tests to like an overkill, maybe testing that you see all pages links is a better test

Reply:
these tests were removed in https://github.com/AgileVentures/WebsiteOne-FE/pull/45